### ISSUE TITLE
No download links found - Repeating Problem

### ISSUE THREAD

**Issue by ZetsApocNow** [Prudhvi-pln/udb#16]
I'm having a problem that I've had before and had to stop using UDB.

Yesterday I downloaded a lot of films and tv series. Today no download links can be found on anything.

I tried to download any film or series, no download links found
I successfully downloaded Charmed S1 yesterday so I check this, no download links found

![image](https://github.com/Prudhvi-pln/udb/assets/93975191/f5a4c5a4-3856-4224-808d-8b953e9cc1c9)

Tried a film I've successfully downloaded yesterday - Papillon 1973, no download links found

![image](https://github.com/Prudhvi-pln/udb/assets/93975191/c9f5e3bd-e6d8-4c46-8540-64835bd6dfa1)

You previously messaged it was a network issue with the src site being blocked, yet I can visit the site fine, enter the film or tv series and watch it without problems.

![image](https://github.com/Prudhvi-pln/udb/assets/93975191/4c980a9d-fd01-4eb4-875c-693c7adbcab8)

I've even tried using a VPN to change my IP to another country

![image](https://github.com/Prudhvi-pln/udb/assets/93975191/135484d6-854f-4446-b127-2d3ed3871635)

This doesn't look like a network issue, as this happens no matter the IP I use, and the source website works fine. This is a repeating issue.

Is it possible I'm facing an API limitation?

**Comment by Prudhvi-pln** [comment]
@ZetsApocNow, Hi there, looks like the current source (vidsrc) updated their API. I am also not able to get download links which was working till date. Instead of fixing this recurring issue, I am planning to move to a different server altogether.

I will update you once it is complete. Please hang in there.

**Comment by ZetsApocNow** [comment]
ok thanks, will do

**Comment by Prudhvi-pln** [comment]
Vidsrc is now restored. It was an encryption issue. However, I'll still proceed with the plan to add a new source.

**Comment by Prudhvi-pln** [comment]
Closing this as vidsrc is restored. There is a separate issue #15 for tracking support for other sources.
